src/topology/algebra/infinite_sum.lean

```diff
@@ -1362,6 +1366,18 @@ begin
   abel,
 end
 
+lemma has_sum.sum_nat_of_sum_int' [t2_space α] {f : ℤ → α} (hf : has_sum f a) :
```

Comment:
Do you really need a `t2_space` assumption here? I expect the lemma to be true in general.

Reply:
Possibly not, but that is true of `has_sum.sum_nat_of_sum_int` too (my fault, I introduced that in #14667). Can we leave that for another PR?

Reply:
I'd rather see it done in this PR; this is the whole point of the review process to get things in a good shape everywhere in the library. Otherwise, I'm pretty sure we will forget about it.